components/Card/Card.js

```diff
@@ -45,7 +45,7 @@ const StyledCard = styled.article`
 
 const StyledBookOverview = styled.div`
   width: 65%;
-  padding: 0px 0px 0px 10px;
+  padding: 0 0 0 10px;
```

Comment:
Suggestion: you can also use `padding-left: 10px` if you only need this one value.

Reply:
Thanks for helping. You're right.